lib/components/collapse.vue

```diff
@@ -43,10 +43,9 @@
         },
 
         watch: {
-            visible(newVal, oldVal){
-                if (newVal !== this.show) {
-                    this.toggle()
-                }
+            visible(newVal) {
+                this.show = newVal;
```

Comment:
Consider this case:
- v-model is `true`
- collapse closes using `v-b-toggle`  (v-model is still `false`)
- v-model sets to `false` => this makes collapse opening again as we are calling toggle :))